services/domino-web-tests/setup.sh

```diff
@@ -23,6 +23,7 @@ cd "${0%/*}"
 
 packages=(
   libgtk-3-0
+  libdbus-glib-1-2
```

Comment:
I think you'll need these too. This is what I need to get past dynamic library loading errors in an empty ubuntu 18.04 container.

```suggestion
  libdbus-glib-1-2
  libgtk-3-0
  libx11-xcb1
  libxt6
```